xcp_d/utils/confounds.py

```diff
@@ -437,6 +443,9 @@ def load_confound_matrix(
         custom_confounds_df = pd.read_table(custom_confounds, sep="\t")
         confounds_df = pd.concat([custom_confounds_df, confounds_df], axis=1)
 
+    confounds_df["linear_trend"] = np.arange(confounds_df.shape[0])
```

Comment:
Does this need to be centered and scaled?

Reply:
I center all of the confounds (except the intercept, of course) after censoring. I figured that would be the way to go, but I can change it to using the full confounds if that's better.

Reply:
Your approach is better, I forgot about the possibility of censoring